iotdb-core/ainode/ainode/core/manager/model_manager.py

```diff
@@ -42,63 +51,194 @@
 class ModelManager:
     def __init__(self):
         self.model_storage = ModelStorage()
+        self._model_status_cache = {}  # Cache for model statuses
+        self._status_lock = ReadWriteLock()
```

Comment:
Currently, the ConfigNode is in charge of the model status, while the AINode only store the model weights and utlize models. It's true that we might optimize this situation, yet this modification should not be this PR. Therefore, plz check and remove all status handle logic